addons/payment/views/payment_form_templates.xml

```diff
@@ -408,37 +408,68 @@
 
     <template id="payment.no_pms_available_warning">
         <div class="alert alert-warning mt-2">
-            <strong>No suitable payment option could be found.</strong>
-            <!-- === Availability report button (for debug) === -->
-            <t t-call="payment.availability_report_button"/>
+            <strong>No payment method available for your order.</strong>
+            <!-- === Availability report button === -->
             <br/>
-            If you believe that it is an error, please contact the website administrator.
+            <t t-if="request.env.is_admin()">
+                <t t-if="providers_sudo">
+                    None is configured for
+                    <t t-out="request.env.company.country_id.name"/>,
+                    <t t-out="request.env.company.currency_id.name"/>.
+                </t>
+                <t t-else="">
+                    You didn't configure any payment provider.
+                </t>
+            </t>
+            <t t-else="">
+                If you believe that it is an error, please contact the website administrator.
+            </t>
             <div class="mt-2" groups="base.group_system">
                 <a
-                    t-if="request.env.company.country_id.is_stripe_supported_country"
+                    t-if="request.env.company.country_id.is_stripe_supported_country
+                        and not providers_sudo"
```

Comment:
When breaking the line in Python expressions, try to align the continuation line for better readability:
```suggestion
                    t-if="request.env.company.country_id.is_stripe_supported_country
                          and not providers_sudo"
```
Optionally, don't break the line in Python expressions if you think that it's more readable.